executionlayer/execution-layer.go

```diff
@@ -761,3 +764,51 @@ func (e *CachingExecutionLayer) GetRPInfo(pubkey rptypes.ValidatorPubkey) (*RPIn
 func (e *CachingExecutionLayer) REthAddress() *common.Address {
 	return e.rEth.Address
 }
+
+// EIP1271ABI is the ABI for the EIP-1271 isValidSignature function
+var eip1271ABI *abi.ABI
+
+// init initializes the EIP1271ABI at program startup
+func init() {
```

Comment:
instead of adding `init` and requiring us to call it on startup let's do the parsing in `getEIP1271ABI` iff `eip1271ABI == nil`.

the `init` pattern is good for things that can fail at runtime due to external dependencies, but since this failure would be exposed by CI, it's safe to do it lazily, imo.

Reply:
Ok, that tracks. Good point that CI would fail _if_ there's a problem.

So as long as CI passes, we can safely assume that this will _not_ panic at runtime, even if this function isn't called for a "long time" after the service starts up?